tests/test_utils.py

```diff
@@ -68,11 +73,19 @@ def test_get_audio_numpy_resample():
         fs=24000,
     )
     wavs_dir = os.path.join(data_dir, "3D-MARCo Impulse Responses/01_Speaker_+90deg_3m")
-    wavs = os.listdir(wavs_dir)
+    wavs = [
+        wavs_dir + "/+90deg_010_OCT3D_1_FL.wav",
+        wavs_dir + "/+90deg_011_OCT3D_2_FR.wav",
+        wavs_dir + "/+90deg_012_OCT3D_3_FC.wav",
+        wavs_dir + "/+90deg_013_OCT3D_4_RL.wav",
+        wavs_dir + "/+90deg_014_OCT3D_5_RR.wav",
+        wavs_dir + "/+90deg_015_OCT3D_6_FLh_1m.wav",
+        wavs_dir + "/+90deg_016_OCT3D_7_FRh_1m.wav",
+        wavs_dir + "/+90deg_017_OCT3D_8_RLh_1m.wav",
+        wavs_dir + "/+90deg_018_OCT3D_9_RRh_1m.wav",
+    ]
     wavs.sort()
-    B = np.array(
-        [librosa.load(os.path.join(wavs_dir, w), 24000, mono=False)[0] for w in wavs]
-    )
+    B = np.array([librosa.load(w, 24000, mono=False)[0] for w in wavs])
```

Comment:
same as before. 

Also, since `wavs` (i.e. the list that will have the form of `['+90deg_010_OCT3D_1_FL.wav', '+90deg_011_OCT3D_2_FR.wav', ...]` will be used more than one test, can we make it a global variable at the top of this file and just call it in the relevant tests? Perhaps `wavs` won't be the best name. Use something more descriptive, since these are OCT-3D files.

Reply:
Good idea. I'll make a global variable called OCT3D_clip_ids, and I'll also make data_dir and wavs_dir global.

Reply:
Cool! Maybe calling it `OCT3D_test_wavs` would be more specific. I agree with the rest. Thanks!